features/meson.build

```diff
@@ -71,6 +71,7 @@ feature_data.set10('_hdr_sys_ldr', cc.has_header('sys/ldr.h', args: feature_test
 feature_data.set10('_hdr_sys_pstat', cc.has_header('sys/pstat.h', args: feature_test_args))
 feature_data.set10('_hdr_sys_ptem', cc.has_header('sys/ptem.h', args: feature_test_args))
 feature_data.set10('_hdr_sys_stream', cc.has_header('stream.h', args: feature_test_args))
+feature_data.set10('_hdr_fts', cc.has_header('fts.h', args: feature_test_args))
```

Comment:
You're not actually using this symbol so why is it needed? Also, we already `#include <fts.h>` where it is needed.

Reply:
Yea, this isn't needed, my bad. Should I add a header check for required include `fts.h`? Otherwise I'll leave only the dynamic library linking fix.